package.json

```diff
@@ -28,40 +28,39 @@
     "test": "ember try:each"
   },
   "dependencies": {
-    "blob-polyfill": "~1.0.20150320",
-    "broccoli-funnel": "^1.2.0",
-    "broccoli-merge-trees": "^1.2.4",
+    "blob-polyfill": "^2.0.20171115",
+    "broccoli-funnel": "^2.0.1",
+    "broccoli-merge-trees": "^2.0.0",
     "broccoli-stew": "^1.5.0",
-    "chalk": "^1.1.1",
-    "ember-cli-babel": "^6.3.0",
+    "chalk": "^2.3.0",
+    "ember-cli-babel": "^6.6.0",
     "ember-cli-string-utils": "^1.1.0",
-    "ember-cli-version-checker": "^1.3.1",
-    "ember-getowner-polyfill": "^1.2.3"
```

Comment:
I think we should leave `ember-getowner-polyfill` for older versions?!

Reply:
Depends on which ember versions is ember-local-storage currently supporting. Only 2.4+ are being tested, and that polyfill is needed only for <2.3, so we should either test e.g. 2.0, or decide that only 2.4+ are supported now and bump major version.

Reply:
Good point. No let's drop support for < 2.3